README.md

```diff
@@ -50,10 +50,14 @@ Example certificates are included in the repository for test purposes.
 
 ```sh
 $ cd quinn
-$ cargo run --example server -- --cert ../certs/server.chain --key ../certs/server.rsa .
-$ cargo run --example client -- --ca ../certs/ca.der https://localhost:4433/Cargo.toml
+$ cargo run --example server -- --cert ./certs/server.chain --key ./certs/server.rsa ./
+$ cargo run --example client -- --ca ./certs/ca.der https://localhost:4433/Cargo.toml
 ```
 
+The above example will work on localhost and serve the "Cargo.toml" file to the client.  If
```

Comment:
This seems slightly misleading, as the server serves the entire directory supplied to it.